File: app/src/main/java/com/surya/shopcart/OTPActivity.java
package com.surya.shopcart;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.widget.TextView;

public class OTPActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_otp);

        TextView loginTextView = findViewById(R.id.createAccount);
        Intent loginActivityIntent = new Intent(this, LoginActivity.class);
        loginTextView.setOnClickListener(view -> {
            startActivity(loginActivityIntent);
        });
    }
}

File: app/src/main/java/com/surya/shopngo/OTPActivity.kt
package com.surya.shopngo

import android.content.Intent
import android.os.Bundle
import android.view.View
import android.widget.TextView
import androidx.appcompat.app.AppCompatActivity

class OTPActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_otp)
        val loginTextView = findViewById<TextView>(R.id.createAccount)
        val loginActivityIntent = Intent(this, LoginActivity::class.java)
        loginTextView.setOnClickListener { view: View? -> startActivity(loginActivityIntent) }
    }
}